### PR TITLE
WIP schemas: refresh authors schema

### PR DESCRIPTION
**Pull request by jacquerie**

### CODE REVIEW

File: inspirehep/dojson/hepnames/fields/bd1xx.py [hunk 1]
```diff
@@ -223,30 +199,6 @@ def native_name2marc(self, key, value):
     }
 
 
-@hepnames.over('private_current_emails', '^595..')
-@utils.for_each_value
-def private_current_emails(self, key, value):
```
Comment (kaplun) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: Ahem, maybe better adding it to the schema.

Comment (kaplun) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: Better put them with other emails with a flag that they should stay hidden.

File: inspirehep/dojson/hepnames/fields/bd1xx.py [hunk 2]
```diff
@@ -361,34 +313,6 @@ def prizes2marc(self, key, value):
     }
 
 
-@hepnames.over('_public_note', '^680..')
-@utils.for_each_value
-def _public_note(self, key, value):
```
Comment (kaplun) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: There are actually public notes. We should add it back to the schema. Actually this is yet a meta-field that is valuable in all schemas.

File: inspirehep/dojson/hepnames/fields/bd1xx.py [hunk 3]
```diff
@@ -361,34 +313,6 @@ def prizes2marc(self, key, value):
...
-
-@hepnames2marc.over('680', '^_public_note$')
-@utils.for_each_value
-def _public_note2marc(self, key, value):
-    return {
-        'i': value
-    }
-
-
-@hepnames.over('_curators_note', '^667..')
```
Comment (kaplun) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: This is a non public note. We should also add it as a meta-field to all schemas. They are useful for cataloguers to write down notes.

**Comment by jacquerie** [comment]
All those fields (except `public_notes`) are not used for searching: they are either metadata useful for cataloguers or data that we want to be hidden. Adding those means dealing with new Sentry issues due to the inevitable crappiness of the data, pushing back the "Enabling Search" milestone. They belong to the "All Editing on Labs" milestone, so I believe we should deal with them later.

**Comment by jacquerie** [comment]
Oh, and `public_notes` should be dealt once and for all in all collections that need it in `inspirehep/dojson/common/base.py`.

File: inspirehep/dojson/hepnames/fields/bd1xx.py [hunk 4]
```diff
@@ -43,30 +43,6 @@
 NON_DIGIT = re.compile('[^\d]+')
 
 
-@hepnames.over('acquisition_source', '^541[10_].')
```
Comment (david-caro) [inline inspirehep/dojson/hepnames/fields/bd1xx.py]: Can you elaborate a bit? (maybe create an issue if not there yet, I tend to forget stuff :older_man: )

**Comment by jacquerie** [comment]
> All those fields (except public_notes) are not used for searching: they are either metadata useful for cataloguers or data that we want to be hidden. Adding those means dealing with new Sentry issues due to the inevitable crappiness of the data, pushing back the "Enabling Search" milestone. They belong to the "All Editing on Labs" milestone, so I believe we should deal with them later.

Let me correct this: I am ok with adding them back to the schema, as long as someone else takes care of correcting the data issues that will pop up on Sentry.

**Comment by jacquerie** [comment]
Mostly superseded by #1596 and #1623

**Pull request closed without merging**